scripts/cover.sh

```diff
@@ -31,7 +31,7 @@ mkdir -p coverage_files
 rm -rf coverage_files/*
 
 echo "Making tests"
-make -j test_unit xfpga
+make -j4 test_unit xfpga
```

Comment:
Why do this?  Did you guys see an issue?

Reply:
Build was failing on virtual memory exhausted. Nakul suspected this was caused by cmake not rationing the resources with -j and suggested this change.